src/core/styles/global.css

```
@@ -0,0 +1,115 @@
+body {
+  @apply min-h-[520px] min-w-[800px] overflow-hidden bg-page font-sans text-sm text-ink;
+}
+
+#root {
+  @apply flex h-full flex-col;
+}
+
+h1,
+h2 {
+  @apply font-display font-semibold tracking-tight;
+  font-variation-settings: "opsz" 24;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)